### DISCUSSION TITLE
Github圖示改變

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#359] [Q&A]
我發現使用windows 11的github discussion的category圖示，與使用windows 10時不一樣。難道作業系統(operating system, OS)的不同，github discussion的category圖示，就會不一樣嗎？

**Comment by JunTingLin** [comment]
可是我在windows 11的github discussion的category圖示跟win10是相同的ㄝ
照理來說作業系統(OS)應該不會影響網頁內容、icon的呈現

## win10
![螢幕擷取畫面 2022-03-02 135513](https://user-images.githubusercontent.com/92431095/156304030-2883fb9e-e5b3-4833-846d-5bf97bee620b.png)
## win11
![螢幕擷取畫面 2022-03-02 134449](https://user-images.githubusercontent.com/92431095/156304066-6f076ff0-b3bd-4215-9d96-084d749f50da.png)
